index.html

```diff
@@ -183,16 +193,17 @@
       </tbody>
     </table>
 
-    <p class="note">The <code>gAMA</code> and embedded ICC profile are provided solely for compatibility with processors that do
-    not conform to this specification.</p>
+    <p class="note">The <code>gAMA</code> chunk, <code>cHRM</code> chunk, and embedded ICC profile specified in the table above are
+    not necessarily accurate, and, even if accurate, their processing is not required to reproduce the image as intended by the
```

Comment:
Ok. cHRM is in fact accurate (it reflects Rec 2020 primaries), so I plan to revise the note to state that ICC and gamma are inaccurate.